students/min/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+class User(models.Model):
+    name       = models.CharField(max_length=100)
+    email      = models.CharField(max_length=200, unique=True)
+    password   = models.CharField(max_length=45)
+    cellphone  = models.CharField(max_length=45)
```

Comment:
```suggestion
    phone_number  = models.CharField(max_length=45)
```
민님 변수명, Field 지정, 컨벤션 다 너무 좋습니다. 하지만 변수명은 명확할수록 좋으니 하나만 수정해주세요!

Reply:
수정 완료 하였습니다!